ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -615,6 +615,16 @@ tests:
         TEST_FUNC=test_olm_operator
       TEST_SUITE: minimal
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-day2
+  optional: true
+  run_if_changed: ^(data/.*)$
```

Comment:
Why data?

Reply:
looks like most optional tests use it, seems reasonable as the test is too long for running oftenly...

Reply:
Running this job when bumping ocp versions is a good idea
Maybe some other directories, I must say I don't really know which ones are relevant specifically for day2